src/containers/userChannels/ChannelsContainer.jsx

```diff
@@ -151,7 +171,7 @@ class ChannelsContainer extends Component {
               render={this.renderPolling}
               pollAction={this.props.pollChannels}
               dueTim={0}
-              periodOfScheduler={2000}
+              periodOfScheduler={5000}
```

Comment:
could you please explain the reason for this change?

Reply:
Yes, basically 2 seconds it's to low for a polling cycle, since we don't want to make this list to be polling all the time i just added a few more seconds to the delay. Also i'm making a first initial request to initialize the channel list so there is no point to have that call and 2 seconds later this one.

Reply:
coolness